data/analysis_oasis.tsv

```diff
@@ -1,3 +1,3 @@
 group	n_subjects	mean_age	std_age	min_age	max_age	sexF	sexM	mean_MMSE	std_MMSE	min_MMSE	max_MMSE	CDR_0	CDR_0.5	CDR_1	CDR_2	CDR_3	mean_scans	std_scans	n_scans
 AD	24.0	75.125	7.321842322803735	62.0	88.0	16.0	8.0	25.458333333333332	3.7636326276026937	15.0	30.0	0.0	16.0	8.0	0.0	0.0	1.0	0.0	24.0
-CN	76.0	41.026315789473685	22.866301155452668	18.0	90.0	51.0	25.0	29.322580645161292	1.0891576139441377	25.0	30.0	31.0	0.0	0.0	0.0	0.0	1.0	0.0	76.0
+CN	17.0	77.05882352941177	9.058441550388196	63.0	90.0	14.0	3.0	29.11764705882353	1.3232025740237636	25.0	30.0	17.0	0.0	0.0	0.0	0.0	1.0	0.0	17.0
```

Comment:
Should this be committed ? It seems like the function you implemented is responsible for doing this modification.

Reply:
Yes, it is the statistical analysis of the dataset in case the user in not able to run the commands.

https://github.com/aramis-lab/clinicadl_handbook/blob/088df1453fc1dd56fee372553d2da7362ae18fe1/src/label_extraction.py#L269-L278

Since patients have been excluded from the dataset, the analysis has changed.